tests/Akkling.Tests/ComputationExpressions.fs

```diff
@@ -34,6 +34,44 @@ let ``Actor computation try-with should generate tail call`` () =  testDefault <
         ref <! i
 
     complete.Task.Wait()
+
+[<Fact>]
+let ``Actor computation try-finally should generate tail call`` () =  testDefault <| fun tck -> 
+    let complete = TaskCompletionSource<_>()
+    let ref = spawn tck "actor" (props <| fun ctx ->
+        let rec loop n =
+            actor {
+                try
+                    let! msg = ctx.Receive()
+                    if msg = 1
+                    then complete.SetResult ()
+                    return! loop msg
+                finally
+                    ()
```

Comment:
We don't have a check if finally block is actually executed, nor if exception thrown within try/finally block will be propagated up.

Reply:
I created the tests and found a few issues.

Please, hold the PR.

Reply:
I'll way for #148 as I need a change from there to move forward with this PR.
I split the PRs to be easier to review them.